resources/views/network/internet/wifi_password.blade.php

```diff
@@ -38,3 +41,23 @@
         </div>
     </div>
 </div>
+
+<script>
```

Comment:
Please add this in a block. That will ensure that the scripts load last, once the document is rendered.
```
@push('scripts')
...
@endpush
```